File: app/src/main/java/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/SilverRing.java
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances;

import it.paranoidsquirrels.idleguildmaster.R;
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.Accessory;

/* JADX INFO: loaded from: classes3.dex */
public class SilverRing extends Accessory {
    @Override // it.paranoidsquirrels.idleguildmaster.storage.data.items.Item
    protected void configureProperties() {
        this.idName = R.string.accessory_silver_ring_name;
        this.idDescription = R.string.accessory_silver_ring_description;
        this.idEffect = R.string.accessory_silver_ring_effect;
        this.idImage = R.drawable.silver_ring;
        this.source.add(Integer.valueOf(R.string.dungeon_name_the_golden_city));
        this.source.add(Integer.valueOf(R.string.raid_name_imperial_rescue));
        this.price = 28L;
        this.initiative = true;
        this.defense = 1;
    }
}

File: app/src/main/kotlin/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/SilverRing.kt
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances

import it.paranoidsquirrels.idleguildmaster.R
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.Accessory

class SilverRing : Accessory() {
    override fun configureProperties() {
        idName = R.string.accessory_silver_ring_name
        idDescription = R.string.accessory_silver_ring_description
        idEffect = R.string.accessory_silver_ring_effect
        idImage = R.drawable.silver_ring
        source.add(R.string.dungeon_name_the_golden_city)
        source.add(R.string.raid_name_imperial_rescue)
        price = 28L
        initiative = true
        defense = 1
    }
}
